OctopusDSC/Examples/cOctopusEnvironment.ps1

```diff
@@ -0,0 +1,18 @@
+Configuration SampleConfig
+{
+    Import-DscResource -Module OctopusDSC
+
+    Node "localhost"
+    {
+        $pass = ConvertTo-SecureString "Password" -AsPlainText -Force
```

Comment:
I suspect this'll fail the Gallery's script analyzer again. They do scan examples. The cOctopusServer example has a different method in it now, using a pre-encrypted password file.